src/main/webapp/profile.jsp

```diff
@@ -0,0 +1,77 @@
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/main.css" type="text/css">
+
+  <script>
+    // scroll the chat div to the bottom
+    function scrollChat() {
+      var chatDiv = document.getElementById('chat');
+      chatDiv.scrollTop = chatDiv.scrollHeight;
+    };
+  </script>
```

Comment:
Should we have a separate js file for scripts, or is leaving this in the jsp okay?

Reply:
I'm leaving it in here because this script is only used twice in the code, but (like the CSS) if it gets redundant we can move it!